BUILD

```diff
@@ -27,11 +27,15 @@ load("//:deployment.bzl", deployment_github = "deployment")
 java_library(
     name = "common",
     srcs = glob([
+        "test/*.java",
+        "test/cluster/*.java",
+        "test/console/*.java",
+        "test/core/*.java",
```

Comment:
do you just want to use `test/**/*.java`?

Reply:
As discussed.